Scripts/Python/grsnNexusBookMachine.py

```diff
@@ -116,6 +108,7 @@ def OnServerInitComplete(self):
         if PtGetPlayerList():
             return
         resetResponder.run(self.key,avatar=PtGetLocalAvatar())
+        bookPillarSpinning.run(self.key, netForce=True, netPropagate=True)
```

Comment:
```suggestion
        bookPillarSpinning.run(self.key)
```

Since you have marked this as `netForce=True` above, these kwargs are redundant.